.changeset/fuzzy-jars-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blitzjs/codemod": patch
+---
+
+fix codemod for wrapping \_app arrow function & fix codemod for nested pages directory
```

Comment:
Can we rephrase it to be more user-facing? Ie what did it change from the user's perspective